tasks/install.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Check if OS-specific variable file exists
```

Comment:
task name should be like **install | Check if OS-specific variable file exists**